numpy/typing/tests/data/reveal/ma.pyi

```diff
@@ -1,10 +1,37 @@
 import numpy as np
 from typing_extensions import assert_type
-m: np.ma.MaskedArray[tuple[int], np.dtype[np.float64]]
+from typing import Any, TypeAlias, TypeVar
+from numpy._typing import _Shape
+from numpy import dtype, generic
 
-assert_type(m.shape, tuple[int])
 
-assert_type(m.dtype, np.dtype[np.float64])
+_ScalarType_co = TypeVar("_ScalarType_co", bound=generic, covariant=True)
+MaskedNDArray: TypeAlias = np.ma.MaskedArray[_Shape, dtype[_ScalarType_co]]
 
-assert_type(int(m), int)
-assert_type(float(m), float)
+class MaskedNDArraySubclass(MaskedNDArray[np.complex128]): ...
+
+MAR_b: MaskedNDArray[np.bool]
+MAR_f4: MaskedNDArray[np.float32]
+MAR_i8: MaskedNDArray[np.int64]
+MAR_subclass: MaskedNDArraySubclass
+MAR_1d: np.ma.MaskedArray[tuple[int], np.dtype[Any]]
+
+b: np.bool
+f4: np.float32
+i8: np.int64
+f: float
```

Comment:
what are these for?

Reply:
pre-empting future work on testing masked array operations, but that was probably premature - i've deleted, thanks!